phoenix-core/src/test/java/org/apache/phoenix/monitoring/PhoenixMetricImplTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.phoenix.monitoring;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class PhoenixMetricImplTest {
+
+    /*
+    Tests the functionality of the TableMetricImpl methods which are exposed
+    1. change()
+    2. increment()
+    3. decrement()
+    4. reset()
+    5. getValue()
+     */
+    @Test public void testTableMetricImplAvailableMethods() {
+
+        PhoenixMetric metric = new PhoenixMetricImpl(MetricType.SELECT_SQL_COUNTER);
```

Comment:
I didn’t see the MetircType class

Reply:
its in https://github.com/vmeka2020/phoenix/blob/PHOENIX-6379/phoenix-core/src/main/java/org/apache/phoenix/monitoring/MetricType.java